charts/rekor/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
 name: rekor
-description: A Helm chart for Kubernetes
+description: Rekkor is an API based server for validation and a transparency log for storage
 
 type: application
 
-version: 0.1.2
+version: 0.1.3
```

Comment:
i would  bump that to 0.2.0 the rekor release is not a bug fix

Reply:
and a small nit: missing the appVersion in this file

Reply:
I added an `appVersion` to the `Chart.yaml` as it separates the lifecycle between the chart and rekor itself